### DISCUSSION TITLE
How to get correct Rotation value for HEIF/HEIC format?

### DISCUSSION THREAD

**Discussion by d2phap** [dlemstra/Magick.NET#775] [Help]
### Description
- Magick.NET returns `1` for Rotation in Exif data for HEIC/HEIF format.
- However, when I use http://exiftool.org/ to extract all Exif profile, the **Rotation value is 270**. 
This information is in the "Quick Time" group actually.

![image](https://user-images.githubusercontent.com/3154213/97953040-4a351180-1dda-11eb-9fe6-9225d419774c.png)

![image](https://user-images.githubusercontent.com/3154213/97953555-dd227b80-1ddb-11eb-9d88-ee28cfd70067.png)

### Steps to Reproduce
- Sample file: [IMG_20201019_171539.zip](https://github.com/dlemstra/Magick.NET/files/5479426/IMG_20201019_171539.zip)
- Get the exif profile:
  ```csharp
  var profile = imgM.GetExifProfile();

  // this line returns 1 (0 degree)
  var exifRotationTag = profile.GetValue(ExifTag.Orientation);   
  ```

### System Configuration
- Magick.NET version: 7.22.1

**Comment by dlemstra** [comment]
You will need to set the define `heic:preserve-orientation` to true if you want to preserve the orientation.
